en/docs/release-notes/version-3.3.0.md

```diff
@@ -21,22 +21,22 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 ## Integrated GraphQL Tryit
 ![graphql-tryit](../../img/release-notes/graphql-tryit.gif)
 
-- You can try out the GraphQL services using the `Tryit` codelens 
+- You can try out the GraphQL services using the `Tryit` CodeLens 
 - You can send and test any request with headers to your GraphQL service in VS Code itself without using any third-party tools
 - The GraphQL explorer will help you to explore the available endpoints and generate the query for you
 
-## Record editor
+## Improved Record Editor
 ![record-editor](../../img/release-notes/record-editor.gif)
 
 - The expression editor is now combined with the record editor, which will give a better editing experience with suggestions and library support
-- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
-- Record creation using JSON samples now supports separate record creation for complex JSON records. 
+- Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation
+- Record creation using JSON samples now supports separate record creation for complex JSON records 
 
-## Project Design view (experimental)
+## Project Design View (experimental)
 ![design-view](../../img/release-notes/design-view.gif)
 
 - This release includes an experimental feature that allows you to visualize service interactions in your project
-- It also comes with a view to see record-type compositions and relationship
-- Use the "Ballerina: Project Design" command to open design view
+- It also comes with a view to see record-type compositions and their relationships
+- Use the `Ballerina: Project Design` command to open the Design View
```

Comment:
```suggestion
- Use the `Ballerina: Project Design` command to open the design view
```

Reply:
UI elements (such as the Record Editor) can be special-cased as pronouns and should be in the title case.

Reply:
Resolving as per https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/19#discussion_r1062672626.